examples/src/main/scala/org/apache/spark/examples/ml/KMeansExample.scala

```diff
@@ -40,20 +42,17 @@ object KMeansExample {
 
     // $example on$
     // Crates a DataFrame
-    val dataset: DataFrame = spark.createDataFrame(Seq(
-      (1, Vectors.dense(0.0, 0.0, 0.0)),
-      (2, Vectors.dense(0.1, 0.1, 0.1)),
-      (3, Vectors.dense(0.2, 0.2, 0.2)),
-      (4, Vectors.dense(9.0, 9.0, 9.0)),
-      (5, Vectors.dense(9.1, 9.1, 9.1)),
-      (6, Vectors.dense(9.2, 9.2, 9.2))
-    )).toDF("id", "features")
+    val rowRDD = spark.read.text("data/mllib/kmeans_data.txt").rdd
```

Comment:
Can you see my comments about this on [#11844](https://github.com/apache/spark/pull/11844/files) and let me know?